lib/src/kbasesearchengine/main/IndexerWorker.java

```diff
@@ -403,35 +407,40 @@ private EventHandler getEventHandler(final String storageCode)
 
     public void processOneEvent(final StatusEvent ev)
             throws IndexingException, InterruptedException, RetriableIndexingException {
+
+        // update event to reflect the latest state of object for which the specified StatusEvent is an event for
+        EventHandler handler = getEventHandler(ev.getStorageCode());
```

Comment:
I think that any exceptions thrown here should be handled correctly, but can you confirm?

Reply:
Yes it is being handled correctly.